evergreen/config.yml

```diff
@@ -535,12 +535,7 @@ tasks:
                 export DEVELOPER_DIR="${xcode_developer_dir}"
             fi
 
-            INSTALL_BAAS_BRANCH=""
-            if [ -n "${baas_branch}" ]; then
-                INSTALL_BAAS_BRANCH="-b ${baas_branch}"
-            fi
-
-            ./evergreen/install_baas.sh -w ./baas-work-dir $INSTALL_BAAS_BRANCH 2>&1 | tee install_baas_output.log
+            ./evergreen/install_baas.sh -w ./baas-work-dir -b master 2>&1 | tee install_baas_output.log
```

Comment:
Are we sure we want to grab master? I guess there's no `stable` branch that we could pull from. Is there any concern about grabbing breaking changes before they're released?

Reply:
nvm - just read the comment about switching back.